### ISSUE TITLE
Change images action box

### ISSUE THREAD

**Issue by huguesvincent** [Automattic/wp-calypso#35782]
In the future, send to Media manager, but add some tooltips building a walkthrough so users are guided through adding images and assigning them to certain places.

**Comment by pablinos** [comment]
Ah, I put this in 'to do', should we split this out as I think the link needs updating as a first step?

**Comment by huguesvincent** [comment]
Actually, I think this should be re-written. 
We should send to the media manager and in a V2 look into a tour guide. 
Isn't this what we agreed on?

**Comment by pablinos** [comment]
Yes I think it is. Let's add a separate issue for the tour guide and use this for the link update.

To be clear, we want 'Change Images' to go to `/media/<SITE URL>`?

**Comment by huguesvincent** [comment]
Yes.
